tests/kernels/conftest.py

```diff
@@ -5,14 +5,9 @@
 
 
 def create_kv_caches(
-    num_blocks: int,
-    block_size: int,
-    num_layers: int,
-    num_heads: int,
-    head_size: int,
-    dtype: torch.dtype,
-    seed: int,
-) -> Tuple[List[torch.Tensor], List[torch.Tensor]]:
+        num_blocks: int, block_size: int, num_layers: int, num_heads: int,
+        head_size: int, dtype: torch.dtype, seed: int,
+        device: str) -> Tuple[List[torch.Tensor], List[torch.Tensor]]:
```

Comment:
Could you append a comma to `device` so that we can minimize the code change? Same for the other places, too.